test/resources/db_test_data.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:mobileprism/services/database/album_data_entry.dart';
+import 'package:mobileprism/services/database/photo_data_entry.dart';
+
+final testAlbum1 = AlbumDataEntry(
+  uid: "1",
+  title: "title",
+  thumbUid: "1",
+);
+final testAlbum2 = AlbumDataEntry(uid: "2", title: "title", thumbUid: "2");
+
+final testPhoto1 = PhotoDataEntry(
+  uid: "1",
+  panorama: true,
+  height: 1,
+  width: 10,
+  lat: 35.87,
+  long: 40,
+  imageHash: "hash",
+  imageQuality: "high_qal",
```

Comment:
This is interesting. 
I thought we store the different `PhotoFormat` as string. But this approch is better.  It might be even better if we later store an integer from 1 to 5 or higher later for all available quality settings: 
```dart  
fit_7680,
fit_2048,
fit_2560,
fit_1280,
fit_720,
  ```
Or we don't even store this and make it a global setting.